perma_web/perma/admin.py

```diff
@@ -134,6 +137,25 @@ def queryset(self, request, queryset):
             return queryset.filter(name__icontains=value)
 
 
+class IAItemFilter(InputFilter):
+    parameter_name = 'item'
+    title = 'IA Item'
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value is not None:
+            return queryset.filter(item_id=value)
+
+class IALinkIDFilter(InputFilter):
+    parameter_name = 'guid'
+    title = 'GUID'
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value is not None:
```

Comment:
Oh. I may be copy/pasting an existing [bug](https://github.com/harvard-lil/h2o/pull/1552)